src/test/java/com/academy/ui/addClub/AddClubPopUpWithManagerTest.java

```diff
@@ -32,13 +35,18 @@ public class AddClubPopUpWithManagerTest extends LoginWithManagerTestRunner {
     private AddClubPopUpStepTwo stepTwo;
     private AddClubPopUpStepThree stepThree;
     private SoftAssert softAssert;
+    private String image1FileName= "image.png";
+    private String image2FileName= "image2.png";
+    private WebDriverWait wait;
+
     private AddClubPopUpSider sider;
 
     @BeforeMethod
     public void addClubPopUpTestPrecondition() {
         addClubPopUpComponent = homePage.header.addClubButtonClick();
         stepOne = addClubPopUpComponent.getStepOneContainer();
         softAssert = new SoftAssert();
+        wait = new WebDriverWait(driver, Duration.ofSeconds(30));
```

Comment:
Why did you add here wait?
Oksana wrote wait methods. You can find them here UploadedImgComponent.java

Reply:
It's Juliya method. I merged from her branch. I don't use it in my TUA-922